www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -446,7 +456,7 @@ function (declare, domClass, dom, style, fx, Toggler, on, query, Memory,
       var that = this;
 
       setTimeout(function () {
-        that.editor.highlightBugPathEvent(that.reportData.lastBugPosition);
+        that.editor.highlightBugLine(that.reportData.line, that.reportData.column);
```

Comment:
You can use the `highlightBugPathEvent` function and the `highlightBugLine` function can be removed.
```js
this.highlightBugPathEvent({
  startLine : lineNum, startCol : column,
  endLine   : lineNum, endCol   : column
});
```

Reply:
I would prefer a separate function, but it is used only at one place and `highlightBugPathEvent` is doing almost the same thing. @bruntib what do you think?